src/pages/Offer/sections/HedvigSwitch.tsx

```diff
@@ -101,20 +125,42 @@ const cols: ReadonlyArray<{
   },
 ]
 
-export const HedvigSwitch: React.SFC = () => (
+interface Props {
+  currentInsuranceState: CurrentInsuranceState
+}
+
+const getOfferSwitchTitle = (currentInsuranceState: CurrentInsuranceState) => {
+  return Object.keys(SwitchableInsurers).includes(
+    currentInsuranceState.currentInsurer!,
+  )
+    ? 'OFFER_SWITCH_TITLE'
+    : 'OFFER_SWITCH_TITLE_NON_SWITCHABLE'
+}
+
+const getColumns = (currentInsuranceState: CurrentInsuranceState) => {
```

Comment:
we could make an implicit return here :) Just something like `(...) => Object.keys(...)....` instead of declaring the full method body